index.html

```diff
@@ -7,4 +7,4 @@
 {% include sections/resources.html %}
 {% include sections/quotes.html %}
 {% include sections/privacy-resources.html %}
-{% include sections/participate.html %}
+{% include sections/participate.html %}
```

Comment:
```suggestion
{% include sections/participate.html %}

```

This is not good style.

Reply:
It does seem to make any difference to the visual side.

I'll commit suggestion.